docs/library/cn0363/index.rst

```diff
@@ -0,0 +1,13 @@
+.. _cn0363_lib:
+
+CN0363 Specific IPs
+================================================================================
+
```

Comment:
Maybe add summary of what is a CN0363 and their IPs?
I would drop the Specific IPs from the title